tests/hardhat/Fork/liquidation.ts

```diff
@@ -355,6 +355,8 @@ if (FORK) {
         if (FORKED_NETWORK == "bsctestnet") repayAmount = 1000000047610436;
         else if (FORKED_NETWORK == "sepolia") repayAmount = 1000000019818620;
         else if (FORKED_NETWORK == "bscmainnet") repayAmount = 1000000034788981;
+        else if (FORKED_NETWORK == "ethereum") repayAmount = 1000000076103691;
```

Comment:
We could extract this to a map

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/360/commits/c0e32f76f0e89ca44b46a0eb0a95ca14d27537a4